quality/jacoco/android.gradle

```diff
@@ -35,19 +35,20 @@ def addJacocoTask(variant) {
   }
 
   def excludedFiles = [
-    '**/R.class', 
-    '**/R$*.class', 
-    '**/BuildConfig.*', 
-    '**/Manifest*.*', 
-    '**/*Test*.*', 
+    '**/R.class',
+    '**/R$*.class',
+    '**/BuildConfig.*',
+    '**/Manifest*.*',
+    '**/*Test*.*',
     'android/**/*.*',
     '**/AutoValue_*.*',
     '**/AutoParcel_*.class',
     '**/*JavascriptBridge.class'
   ]
 
-  jacocoTask.sourceDirectories = files(variant.sourceSets.java.srcDirs.flatten())
-  jacocoTask.classDirectories = fileTree(dir: variant.javaCompile.destinationDir, excludes: excludedFiles)
+  jacocoTask.sourceDirectories = files(variant.sourceSets.java.srcDirs.flatten(), "src/main/kotlin")
+  jacocoTask.classDirectories = fileTree(dir: variant.javaCompile.destinationDir, excludes: excludedFiles) +
+    fileTree(dir: "$buildDir/tmp/kotlin-classes/${variant.name}", excludes: excludedFiles)
```

Comment:
i think we should only do these additions if the kotlin plugin exists, so better wrap this in a `if` condition.
Moreover `variant.sourceSets.kotlin.srcDirs.flatten()` is better imo (your path probably won't work on windows).
More importantly: can we get the kotlin output classes in a more robust way? is there a way to get hold of something like `variant.kotlinComplie`? because build dir layouts tend to change, so I expect this to break between different versions of kotlinc.

Reply:
I agree for the `if`. I will change it.

The problem with `variant.sourceSets.kotlin.srcDirs.flatten()` is that the `kotlin` property does not exist in `sourceSets`. And I didn't find a way to get the output class in a more robust way. 
I will continue to search a better way. 

Also I'm not really familiar with gradle script and sourceSet, so if you have any advice, please tell me.